openapi.json

```diff
@@ -18767,6 +19668,186 @@
                 ]
             }
         },
+        "/api/v1/transformation-template/generate": {
```

Comment:
I wonder if this route should have a trailing `/` to match the rest?

Sorry to only notice this now 😭

Reply:
😮 you're right. I think I missed that one in my PR. I'll merge this one and fix that in a follow up.